crates/router/src/types/api/payments.rs

```diff
@@ -19,13 +19,15 @@ pub use api_models::payments::{
     VerifyResponse, WalletData,
 };
 #[cfg(feature = "v2")]
-pub use api_models::payments::{PaymentsCreateIntentRequest, PaymentsIntentResponse};
+pub use api_models::payments::{
```

Comment:
we should ideally not have re-exports like this, see if this can be removed for others as well

Reply:
Will take up separately